### PR TITLE
ci: Bump timeout for Postgres CDC tests

### PR DESCRIPTION
**Pull request by def-**
Seen in coverage in https://buildkite.com/materialize/coverage/builds/168#01892500-083b-400e-88c9-1320b6683a3d

### Checklist

- [ ] This PR has adequate test coverage / QA involvement has been duly considered.
- [ ] This PR has an associated up-to-date [design doc](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/design/README.md), is a design doc ([template](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/design/00000000_template.md)), or is sufficiently small to not require a design.
  
- [ ] If this PR evolves [an existing `$T ⇔ Proto$T` mapping](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/command-and-response-binary-encoding.md) (possibly in a backwards-incompatible way), then it is tagged with a `T-proto` label.
- [ ] If this PR will require changes to cloud orchestration, there is a companion cloud PR to account for those changes that is tagged with the release-blocker label ([example](https://github.com/MaterializeInc/cloud/pull/5021)).
  
- [ ] This PR includes the following [user-facing behavior changes](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/guide-changes.md#what-changes-require-a-release-note):
  -

### CODE REVIEW

File: ci/test/pipeline.template.yml [hunk 1]
```diff
@@ -387,7 +387,7 @@ steps:
   - id: pg-cdc
     label: Postgres CDC tests
     depends_on: build-x86_64
-    timeout_in_minutes: 30
+    timeout_in_minutes: 45
```
Comment (philip-stoev) [inline ci/test/pipeline.template.yml:390]: The test runs in 8 min without Coverage, so this value was more than sufficient.

**Review by nrainer-materialize — APPROVED** [review]

File: ci/test/mkpipeline.py [hunk 2]
```diff
@@ -99,7 +99,7 @@ def main() -> int:
         for step in pipeline["steps"]:
             # Coverage runs are slower
             if "timeout_in_minutes" in step:
-                step["timeout_in_minutes"] *= 2
+                step["timeout_in_minutes"] *= 3
```
Comment (philip-stoev) [inline ci/test/mkpipeline.py:102]: You are welcome to bump this to 4 , to be safe.

**Comment by philip-stoev** [comment]
Unfortunately the output seems to indicate that the postgres source was truly stuck for 1 full hour. The last successfull test **with coverage** took 10 min. There are quite a few segfaults in the same CI job, so I think something broke in a more general way.

**Comment by def-** [comment]
The segfaults are unrelated, I think: https://github.com/MaterializeInc/materialize/issues/20016

**Pull request closed without merging**